ui/src/org.ts

```diff
@@ -433,33 +429,35 @@ export async function fetchMembers(
     gnosisSafeAddress
   );
 
+  return {
+    threshold: response.threshold,
+    members: response.members.map(address => {
+      return { ethereumAddress: address };
+    }),
+  };
+}
+
+export async function resolveMemberIdentities(
+  unresolvedMembers: Member[]
```

Comment:
As per my comment above, this would become
```suggestion
  addresses: string[]
```